tutorials/transfer_learning/dataloader.jl

```diff
@@ -7,16 +7,41 @@ if isempty(readdir(PATH))
   error("Empty train folder - perhaps you need to download and extract the kaggle dataset.")
 end
 
+# Get all of the files with "dog" in the name
 const DOGS = filter(x -> occursin("dog", x), FILES)
+
+# Get all of the files with "cat" in the name
 const CATS = filter(x -> occursin("cat", x), FILES)
 
+# Takes in the number of requested images per batch ("n") and image size
+# Returns a 4D array with images and an array of labels
 function load_batch(n = 10, nsize = (224,224); path = PATH)
+  if ((batchsize % 2) != 0)
```

Comment:
Is that really the case?

Reply:
Yes, it errors if you pass in 1 or 3 for example

Reply:
Maybe we need to use `\div` instead of `/` in line 23 (`img_paths = ...`)

Reply:
The same error occurs even when using that division sign